sql/core/src/main/scala/org/apache/spark/sql/execution/Columnar.scala

```diff
@@ -96,9 +96,6 @@ case class ColumnarToRowExec(child: SparkPlan)
   /**
```

Comment:
`ColumnarToRowExec`'s comment also mentions `ColumnarBatchScan`. If you are like to remove all reference to `ColumnarBatchScan`...

Reply:
Great catch. I thought I got rid of all of them.  Will grep though again.